Alameen688/MyDiary
# [Chore #159365053] Deliver action on feedback

#### What does this PR do?
Delivers feedback made by LFA during end of week 1 review
#### Description of Task to be completed?
The following should feedback must have been worked on
-Prevent posting of unnecessary white spaces in body of request by trimming
- Keep forms within viewport height
- Login page doesn't need navigation button for login
- Signup page doesn't need navigation button for signup
- Include reference in UI template
#### How should this be manually tested?
- send request body data with trailing whitespaces on any api endpoint, if succes=sul whitespace should be trimmed

Go to
 - https://alameen688.github.io/MyDiary/UI/login.html
 - https://alameen688.github.io/MyDiary/UI/signup.html
After cloning the repo
 - check /css/style.css and css/auth.js for references
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
#159365053
